src/test/resources/groups-testng1.xml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
You don't have any groups defined for your tests so you either define some groups for your tests and use them in this .xml file or you remove this file for groups. Thanks!